api-gateway/api-gateway-kubernetes.yml

```diff
@@ -12,10 +12,17 @@ spring:
         - id: profilesDataPrincipal
           uri: http://profiles
           predicates:
-            - Path=/api/v1/influencers/me/followersCount, /api/v1/influencers/me/externalReferences
```

Comment:
why the followers count was removed?

Reply:
this operation was moved to the social media adapter.

Reply:
social media adapter should perform a call to the campaign manager to save the actual number of followers in our db:
https://agena-tech.atlassian.net/wiki/spaces/INF/pages/19890200/Set+Followers+Count

the endpoint above should be used for that